recipe/meta.yaml

```
@@ -1,47 +1,53 @@
-{% set name = "radio_beam" %}
+{% set name = "radio-beam" %}
 {% set version = "0.3.9" %}
 
 package:
-  name: radio-beam
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace("-", "_") }}-{{ version }}.tar.gz
   sha256: 9b5fea7bcc9b26543213784633b32e81630c9c0855077b7abf4b46cf8d84e644
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-deps -vv
 
 requirements:
-
   host:
-    - python {{ python_min }}
     - pip
+    - python
+    - setuptools
```

https://github.com/radio-astro-tools/radio-beam/blob/v0.3.9/tox.ini#L7
```suggestion
    - setuptools >=30.3.0
```